front_end/entrypoints/inspector_main/RenderingOptions.ts

```diff
@@ -213,6 +219,9 @@ export class RenderingOptionsView extends UI.Widget.VBox {
     this.appendCheckbox(
         i18nString(UIStrings.paintFlashing), i18nString(UIStrings.highlightsAreasOfThePageGreen),
         Common.Settings.Settings.instance().moduleSetting('showPaintRects'));
+    this.appendCheckbox(
```

Comment:
Don't we duplicate this in the `CohtmlPanel` as well?

Reply:
yep. I'll remove it from here